cloud/pkg/cloudstream/tunnelserver.go

```diff
@@ -79,6 +80,18 @@ func (s *TunnelServer) getSession(id string) (*Session, bool) {
 	return sess, ok
 }
 
+func (s *TunnelServer) addNodeIP(node, ip string) {
+	s.nodeNameIP.Store(node, ip)
+}
+
+func (s *TunnelServer) getNodeIP(node string) (string, bool) {
+	ip, ok := s.nodeNameIP.Load(node)
+	if !ok {
+		return "", ok
+	}
+	return ip.(string), ok
```

Comment:
those check is redundant, you have checked the bool in the place where it is got called.

Reply:
if not check，ip.(string) will get a panic.